docs/_static/crypto/v1.yaml

```diff
@@ -62,7 +62,70 @@ paths:
                 example: "aa==.bb==.cc=="
         default:
           $ref: '../common/error_response.yaml'
+  /internal/crypto/v1/encrypt_jwe:
+    post:
+      summary: "encrypt a payload and headers with the public key of the given DID into a JWE object"
+      description: |
```

Comment:
Can you link to the RFC. What is supported, what not. Which encryption algorithms are used (by default).